cmd/stork/stork.go

```diff
@@ -159,6 +159,11 @@ func main() {
 			Value: 10,
 			Usage: "The interval in seconds to sync reconcilers (default: 10 seconds)",
 		},
+		cli.IntFlag{
+			Name:  "k8s-api-rate-limiter",
+			Value: 100,
+			Usage: "Restrict number of k8s api requests from stork (default: 100 QPS, 100 Burst)",
```

Comment:
should we allow to configure QPS and Burst separately?

Reply:
Yes please. Lets separate these out - `k8s-api-qps` & `k8s-api-burst`